wgpu/src/lib.rs

```diff
@@ -3119,7 +3120,7 @@ impl<'a> RenderPass<'a> {
         );
     }
 
-    /// Sets the viewport region.
+    /// Sets the viewport region of framebuffer space.
```

Comment:
I don't think "framebuffer" is all that well defined? Not even sure if the spec ended up needing a color target now on renderpass or not..

Could just go with what WebGPU spec writes: `Sets the viewport used during the rasterization stage to linearly map from [normalized device coordinates](https://www.w3.org/TR/webgpu/#ndc) to [viewport coordinates](https://www.w3.org/TR/webgpu/#viewport-coordinates).`

Reply:
This still needs to be changed to "viewport coordinates" as that's the spec name for it.

Reply:
sorry this was updated i just did not get a chance to push it XD. please check it again.